cribl-edge/package/clean.sls

```diff
@@ -0,0 +1,16 @@
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+
+{%- from tplroot ~ "/map.jinja" import mapdata as cribl_edge with context %}
+
+Cribl-Edge App-Dir Removed:
+  file.absent:
+    - name: {{cribl_edge.package.install_path }}
+    - onlyif:
+      - '[[ -d {{cribl_edge.package.install_path }} ]]'
```

Comment:
I don't think the onlyif is needed for the clean right? That check was only needed when creating the logdir setup, to stop it from deleting the symlink. For this, the statefulness of `file.absent` should be enough?

```suggestion
```

Reply:
Fixed